java_src/foam/nanos/boot/NSpec.java

```diff
@@ -0,0 +1,143 @@
+// DO NOT MODIFY BY HAND.
```

Comment:
This shouldn't be checked in. Should only be generated as part of the build and then discarded.